### PR TITLE
Add Louis-Guillaume Gagnon to team

### PR DESCRIPTION
**Pull request by gagnonlg**
Add myself to the fellows list, and also submit a fellows project.

### CODE REVIEW

File: pages/projects/acts.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ team:
  - rockybala
  - Tomohiro Yamazaki
  - Irina Ene
+ - Louis-Guillaume Gagnon
```
Comment (henryiii) [inline pages/projects/acts.md]: These should be your shortname (gagnonlg).

File: _data/fellow_projects/acts-algs-ml-gpu.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+open: yes
+postdate: 2021-03-18
+title: "**Efficient implementation of algorithms to reconstruct charged particles trajectories**"
+description: "Reconstruction of charged particle trajectories in a magnetic field is a common technique used by many high-energy physics experiments to estimate the momentum of produced particles.  [ACTS](https://acts.readthedocs.io/en/latest/) is a software library aiming to provide cross-experiment support for charged track reconstruction algorithm. Written in modern C++, ACTS is designed to be as efficient as possible while still allowing maximum adaptability to the different needs of experiment-specific software. This project aims to improve existing algorithms and implement new, cutting edge techniques including machine learning models and GPU-accelerated algorithms within the ACTS framework."
+contacts:
+ - Louis-Guillaume Gagnon (gagnonlg@lbl.gov)
```
Comment (henryiii) [inline _data/fellow_projects/acts-algs-ml-gpu.yaml]: Same here, should be shortname.

**Comment by henryiii** [comment]
From the logs: ` _data/people/gagnonlg.yml must contain title`

It can be (and usually is) empty, but currently needs to be there. I might be able to fix that tomorrow.

**Comment by gagnonlg** [comment]
Thanks @henryiii , should be fixed now

**Pull request merged**